styles.css

```diff
@@ -38,6 +38,38 @@ body {
   background-color: var(--surface-primary);
 }
 
+.header__title {
+  color: var(--text-primary);
+  font-family: "Exo 2";
+  font-size: 3rem;
+  font-weight: 700;
+  line-height: 3.5rem;
+  margin: 1rem;
+  text-align: center;
+}
+
+.header__paragraph {
+  color: var(--text-secondary);
+  font-family: "Exo 2";
+  font-size: 1rem;
+  font-weight: 300;
+  line-height: 1.5rem;
+  margin: 4rem;
```

Comment:
Ao invés de utilizar `margin` de `4rem` sugiro seguir o layout do figma, onde está `1rem` apenas em cima.

![image](https://user-images.githubusercontent.com/53842126/170570543-5879a418-d12e-453b-ad70-2f18ce092349.png)

```suggestion
  margin-top: 1rem;
```

Reply:
Vou arrumar isso.

Reply:
Não sei como essas mudanças no css pararam aí, acho que foi pela questão da branch errada, não sei.